### ISSUE TITLE
Deprecation warning for broken import on future Python 3.10 

### ISSUE THREAD

**Issue by HelioGuilherme66** [robotframework/PythonRemoteServer#78]
I was just experimenting the Example, and got the following output:
```
C:\Users\Helio>python -m robotremoteserver test
C:\Python39\lib\site-packages\robotremoteserver.py:18: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working
  from collections import Mapping
Remote server running at http://127.0.0.1:8270.

C:\Users\Helio>python --version
Python 3.9.6
```

**Comment by DanielPenning** [comment]
Can confirm for the current pip release (also for python 3.10
However, this was already fixed 2 years ago on master (e625c7470323496f5699441bc535eb314be501fb).

@pekkaklarck Is it easily possible to publish a release containing the above fix?

**Comment by ShemTovYosef** [comment]
Hi @pekkaklarck 
Since robot framework supports it from 4.1.1, we want to move to python 3.10 but missing this fix in PythonRemoteServer stops us.
Any change to get it?

With regards,
Yosef

**Comment by SePossum** [comment]
Hi,

I found the same problem today when setting up a new computer with python 3.10.4 and robotframework 5.0:

The following change fixes the problem for me:

In ...\Python\Python310\Lib\site-packages\robotremoteserver.py
change line 18 
"""
from collections import Mapping
"""
to
"""
try:
	from collections import Mapping
except:
	# for Python 3.10
	from collections.abc import Mapping, MutableMapping
"""

Regards,
Bernd

**Comment by oberi01** [comment]
Hi @pekkaklarck,
same as Yosef: the broken import stops us from moving to Python310.
Any plans of releasing a fix?
Thanks!
Richard

**Comment by Snooz82** [comment]
+1

**Comment by UyttenhoveSimon** [comment]
I concur with the above folks, it would greatly help us if you could push a new release.

**Comment by pekkaklarck** [comment]
This problem has been reported and already fixed in 2020 (#58) but there hasn't been any release after that. A bug fix release with this and other Python compatibility fixes is now planned for next Thursday.

**Comment by pekkaklarck** [comment]
Closing this as a duplicate of #58.

**Comment by pekkaklarck** [comment]
Already fixed #81 covers similar problem with Python 3.11. Need to think should I create v1.1.1 bug fix release with these fixes already this weekend instead of waiting for next Thursday. We could then possibly have v1.2 with other enhancements later.

**Comment by ShemTovYosef** [comment]
> Already fixed #81 covers similar problem with Python 3.11. Need to think should I create v1.1.1 bug fix release with these fixes already this weekend instead of waiting for next Thursday. We could then possibly have v1.2 with other enhancements later.

Thank you , great news! 
Better to provide 1.1.1 so a new  functionality will not affect current projects

**Comment by pekkaklarck** [comment]
If we do 1.2 instead of 1.1.1, it will be a small release and should be safe to upgrade to. It would mainly (only?) add `get_library_information` method (#75) which would speed up Remote library usage considerably especially with bigger libraries.
